lib/web/join_tokens.go

```diff
@@ -314,6 +338,15 @@ func getJoinScript(ctx context.Context, settings scriptSettings, m nodeAPIGetter
 		labelsList = append(labelsList, fmt.Sprintf("%s=%s", labelKey, labels))
 	}
 
+	resourceLabelsList := []string{}
```

Comment:
nit: https://github.com/golang/go/wiki/CodeReviewComments#declaring-empty-slices

I also think the variable name could be a bit more descriptive to differentiate it from `labelsList` and make it clear this is used for the db service label matchers